presto-main/src/main/java/io/prestosql/operator/WindowOperator.java

```diff
@@ -96,7 +98,8 @@ public WindowOperatorFactory(
                 int expectedPositions,
                 PagesIndex.Factory pagesIndexFactory,
                 boolean spillEnabled,
-                SpillerFactory spillerFactory)
+                SpillerFactory spillerFactory,
+                OrderingCompiler orderingCompiler)
```

Comment:
i'd add this param after `int preSortedChannelPrefix,`, or after `PagesIndex.Factory pagesIndexFactory,`